docs/transparent-validation/rte/v1.1.0/quick-start-guide.md

```diff
@@ -0,0 +1,51 @@
+# Quick start guide
+
+The canonical example of RTE usage is hooking it to some hardware for SPI
+flashing, power control and serial logs gathering. This document describes the
+common preparation of RTE without listed functions.
+
+## Prerequisites
+
+* [RTE board](https://3mdeb.com/shop/open-source-hardware/open-source-hardware-3mdeb/rte/)
+    (this document is based on v1.1.0)
+* Micro-USB 5V 2.5A power supply
+* Orange Pi
+* SD card
+* Ethernet cable
+* USB-UART converter with 3 wire cables
+* Ubuntu (based on 22.04)
+
+## Preparation of RTE
+
+1. Build RTE image using the
+    [meta-rte repository](https://github.com/3mdeb/meta-rte).
+1. Flash SD card using this image. The flashing method isn't imposed but an
```

Comment:
Why do we recommend `etcher`? In my opinion, the better option is to re-write this point similar to the point about flashing in the `SD-wire` documentation.